modules/ducktests/tests/README.md

```diff
@@ -0,0 +1,14 @@
+## Overview
+The `ignitetest` framework provides basic functionality and services
+to write integration tests for Apache Ignite. This framework bases on 
```

Comment:
s/bases on/is based on/

Reply:
I agree and tried use passive voice but Intellij Idea recommends use active voice here.